samples/msgext-doc-compliance-checker/README.md

```diff
@@ -0,0 +1,102 @@
+---
+page_type: sample
+description: This sample demonstrates how to integrate Azure AI Search in a Teams message extension to compare a contract proposal against a guidance policy checklist to see if the proposal complies with the guidance. It uses Teams Toolkit for Visual Studio Code and JavaScript, and the message extension can be used as a plugin in Microsoft Copilot for Microsoft 365.
+products:
+- office-teams
+- copilot-m365
+languages:
+- javascript
+---
+
+# Compliance Checker with Azure AI Search sample
+
+## Interaction with app
+
+ ![msgext-doc-compliance-checkerGif](Images/msgext-doc-compliance-checkerGif.gif)
+
+## Try it yourself - experience the App in your Microsoft Teams client
```

Comment:
We are not gonna try in Teams client only. It should be tested in Teams Copilot as well so update at all the places.

Reply:
Fixed